osu.Game/Screens/Play/HUD/FaillingLayer.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    /// <summary>
+    /// An overlay layer on top of the playfield which fades to red when the current player health falls a certain threshold defined by <see cref="LowHealthThreshold"/>.
+    /// </summary>
+    public class FaillingLayer : HealthDisplay
```

Comment:
```suggestion
    public class FailingLayer : HealthDisplay
```
same deal in usages, etc.